### PR TITLE
Added wish list api

### PR DESCRIPTION
**Pull request by ankit4help**

### CODE REVIEW

File: src/stubs/wishlistService.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                body: JSON.stringify(successResponse)
+            }
+
+        }]
+
+
+
+
+
+
```
Comment (iamnitinkumar) [inline src/stubs/wishlistService.js]: Remove so many blank lines

File: src/stubs/wishlistService.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            "statusCode": 404,
+            "headers": {
+                "connection": "close"
+            },
+            body: JSON.stringify(serviceNotFoundResponse)
+          }
+    };
+
+
+
```
Comment (iamnitinkumar) [inline src/stubs/wishlistService.js]: Remove so many blank lines

File: src/stubs/wishlistService.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    return mbHelper.postImposter(imposter);
+
+
```
Comment (iamnitinkumar) [inline src/stubs/wishlistService.js]: Remove so many blank lines

**Review by iamnitinkumar — CHANGES_REQUESTED** [review]

**Comment by ankit4help** [comment]
removed all blank lines

**Review by iamnitinkumar — APPROVED** [review]

**Pull request merged**